src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java

```diff
@@ -1,7 +1,59 @@
 package com.zipcodewilmington.phonebook;
 
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.TreeMap;
+
 /**
- * Created by leon on 1/23/18.
+ * Created by Mitch not Leon 2/14/18
  */
+
+
 public class PhoneBook {
-}
+
+    private TreeMap<String, ArrayList<String>> phoneBookList = new TreeMap<String, ArrayList<String>>();
+
+    public void addEntryToPhoneBook(String name, String... phonenumbers) {
+        ArrayList<String> phonenumberList = new ArrayList<String>();
+        for (String number : phonenumbers) {
+            phonenumberList.add(number);
+        }
+        phoneBookList.put(name, phonenumberList);
+    }
+
+    public void removeEntryFromPhoneBook(String name) {
+        phoneBookList.remove(name);
+    }
+
+    public String listAllNames() {
+        return phoneBookList.keySet().toString();
+    }
+
+    public String listAllEntries() {
+        return phoneBookList.entrySet().toString();
+    }
```

Comment:
NQR: you need to fix your list methods, they should print out nice lists that could be printed for use?